packages/agent-simulation/src/components/agent-simulation.tsx

```diff
@@ -24,6 +24,12 @@ import css from "./agent-simulation.scss";
 
 interface IProps extends IRuntimeQuestionComponentProps<IAuthoredState, IInteractiveState> {}
 
+type VisHandleWithTicker = ReturnType<typeof AV.vis> & {
+  ticker?: {
+    maxFPS: number;
+  };
+};
```

Comment:
The return value of `AV.vis` is `new PIXI.Application()`. Is there any way to get that type?

Either way, I think you should be able to use `VisHandle` from  `types/atomic-agents-vis.d.ts`. You'll see `VisHandle` has a member `app` that is type `any` there. It would be great to improve the `any`, but it should work even if you don't.